numpy/lib/tests/test_function_base.py

```diff
@@ -3490,6 +3495,15 @@ def test_fraction(self):
         x = np.arange(8)
         assert_equal(np.quantile(x, Fraction(1, 2)), Fraction(7, 2))
 
+    def test_complex(self):
+        #See gh-22652
+        arr_c = np.array([0.5+3.0j, 2.1+0.5j, 1.6+2.3j], dtype=np.dtype('complex256'))
+        assert_raises(TypeError, np.quantile, arr_c, 0.5)
+        arr_c = np.array([0.5+3.0j, 2.1+0.5j, 1.6+2.3j], dtype=np.dtype('complex128'))
+        assert_raises(TypeError, np.quantile, arr_c, 0.5)
+        arr_c = np.array([0.5+3.0j, 2.1+0.5j, 1.6+2.3j], dtype=np.dtype('complex64'))
```

Comment:
The linter is unhappy about these, would be nice to shorten or break.

Reply:
I shortened the lines, but the linter still complains. All of the lines I shortened are now 78 characters long, which is within the PEP 8 guidelines, so the linter might be complaining about something else.

Reply:
Yes, because you only changed half of them, I fixed them up...